src/api/routes.py

```diff
@@ -126,7 +126,7 @@ def handle_user(user_id):
         return response_body, 200
 
 
-@api.route('/ingredients/<int:ingredient_id>', methods=['GET'])
+@api.route('/exercise/<int:ingredient_id>', methods=['GET'])
```

Comment:
@is-stefano 
Porque el endpoints se llama `/exercise/<int:ingredient_id>`

Reply:
@hchocobar pusimos exercise por no borrarlo, porque se supone que íbamos a reutilizarlo para otros endpoints. No vale esa ruta de momento